feconf.py

```diff
@@ -453,6 +453,10 @@ def get_empty_ratings():
 # Mailchimp username.
 MAILCHIMP_USERNAME = None
 
+# If requests can be sent to the bulk email provider (requires the Mailchimp API
+# key, username and audience id fields to be filled).
```

Comment:
Problem is, we aren't enforcing this constraint anywhere, so from the outside this constant just looks like an independent configurable setting...

Maybe sufficient to say something like this?
```
CAN_SEND_REQUESTS_TO_BULK_EMAIL_SERVICE_PROVIDER = bool(
    PROD_MODE and MAILCHIMP_AUDIENCE_ID and MAILCHIMP_API_KEY and MAILCHIMP_USERNAME)
```

Reply:
Done.